tests/PHPStan/Analyser/data/array-types.php

```diff
@@ -8,24 +8,24 @@ class Foo
 {
 
 	/**
-	 * @param array{foo: '1', bar: '2', lall: '3'} $array1
-	 * @param array{foo: '1', bar: '4', lall2: '3'} $array2
+	 * @param array{foo: '1', bar: '2', lall: '3', 2: '2', 3: '3'} $array1
+	 * @param array{foo: '1', bar: '4', lall2: '3', 2: '4', 3: '6'} $array2
 	 */
 	public function arrayFunctions($array1, $array2): void
 	{
-		assertType("array('1' => '1', '2' => '2', '3' => '3')", array_combine($array1, $array1));
-		assertType("array('1' => '1', '2' => '4', '3' => '3')", array_combine($array1, $array2));
+		assertType("array('1' => '1', '2' => '2', '3' => '3', '2' => '2', '3' => '3')", array_combine($array1, $array1));
+		assertType("array('1' => '1', '2' => '4', '3' => '3', '2' => '4', '3' => '6')", array_combine($array1, $array2));
 
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_merge($array1));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_merge([], $array1));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_merge($array1, []));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_merge($array1, $array1));
-		assertType("array<'bar'|'foo'|'lall'|'lall2', '1'|'2'|'3'|'4'>&nonEmpty", array_merge($array1, $array2));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 0 => '2', 1 => '3')", array_merge($array1));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 0 => '2', 1 => '3')", array_merge([], $array1));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 0 => '2', 1 => '3')", array_merge($array1, []));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 0 => '2', 1 => '3', 2 => '2', 3 => '3')", array_merge($array1, $array1));
+		assertType("array('foo' => '1', 'bar' => '4', 'lall' => '3', 0 => '2', 1 => '3', 2 => '3', 3 => '4', 4 => '6')", array_merge($array1, $array2));
 
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_replace($array1));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_replace([], $array1));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_replace($array1, []));
-		assertType("array<'bar'|'foo'|'lall', '1'|'2'|'3'>&nonEmpty", array_replace($array1, $array1));
-		assertType("array<'bar'|'foo'|'lall'|'lall2', '1'|'2'|'3'|'4'>&nonEmpty", array_replace($array1, $array2));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 2 => '2', 3 => '3')", array_replace($array1));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 2 => '2', 3 => '3')", array_replace([], $array1));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 2 => '2', 3 => '3')", array_replace($array1, []));
+		assertType("array('foo' => '1', 'bar' => '2', 'lall' => '3', 2 => '2', 3 => '3')", array_replace($array1, $array1));
+		assertType("array('foo' => '1', 'bar' => '4', 'lall' => '3', 2 => '4', 3 => '6', 'lall2' => '3')", array_replace($array1, $array2));
```

Comment:
Array shape syntax uses `{}` not `()` brakets

see https://phpstan.org/writing-php-code/phpdoc-types#array-shapes

Reply:
I think this is still a cosmetic issue in phpstan, the error message still use array() syntax instead of array{}.

Reply:
I want to sync that for 1.0